client/tailscale/keys.go

```diff
@@ -72,7 +72,10 @@ func (c *Client) Keys(ctx context.Context) ([]string, error) {
 // can be created. Returns the key itself, which cannot be retrieved again
 // later, and the key metadata.
 func (c *Client) CreateKey(ctx context.Context, caps KeyCapabilities) (string, *Key, error) {
-	bs, err := json.Marshal(caps)
+	keyRequest := struct {
```

Comment:
Is this somehow related to the other cleanups or just another necessary change?

Reply:
This is an unrelated bugfix that's necessary for the operator to work, I messed up the implementation of client/tailscale in a PR yesterday. Unrelated to the PR's title though, it was just in my queue and I missed that I was bundling it here.

Reply:
👍